src/xlat-i/xi-Parameter.C

```diff
@@ -712,7 +712,7 @@ void ParamList::beginUnmarshallSDAGCall(XStr& str, bool usesImplBuf) {
     if (hasArray || hasRdma()) {
       if (!usesImplBuf) {
         str << "  genClosure->_impl_marshall = impl_msg_typed;\n";
```

Comment:
These two lines could be combined as well

Reply:
Doing that causes a type error due to it returning a `void*`, so I'm leaving this one as it is.